lib/task-data/base-tasks/copy-key.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+
+module.exports = {
+    friendlyName: 'Copy Key',
+    injectableName: 'Task.Base.Ssh.CopyKey',
+    runJob: 'Job.CopyKey',
+    requiredOptions: [],
```

Comment:
Since you have designed some assertion logic for some SSH options, so it's better to add them into the `requireOptions`, the same for `validate-ssh` base-task.